From 9c94439a5a55efbb72bee10e9cfa0827dbedf511 Mon Sep 17 00:00:00 2001
From: "Jason M. Bills" <jason.m.bills@linux.intel.com>
Date: Mon, 3 Jun 2019 16:35:46 -0700
Subject: [PATCH] Add host interrupt to the Host State

This adds host Interrupt as possible Transition value for the
Host State. This is to support an interrupt signal to the host
firmware or software.

Signed-off-by: Jason M. Bills <jason.m.bills@linux.intel.com>
---
 xyz/openbmc_project/State/Host.interface.yaml | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/xyz/openbmc_project/State/Host.interface.yaml b/xyz/openbmc_project/State/Host.interface.yaml
index 1ceba13..3d71280 100644
--- a/xyz/openbmc_project/State/Host.interface.yaml
+++ b/xyz/openbmc_project/State/Host.interface.yaml
@@ -28,6 +28,9 @@ enumerations:
       description: >
         The desired transition for host firmware
       values:
+        - name: 'Interrupt'
+          description: >
+            Host firmware should be interrupted
         - name: 'Off'
           description: >
             Host firmware should be off
-- 
2.7.4

